src/utils/templates.ts

```diff
@@ -18,7 +18,9 @@ export const generatedContentTemplate = (generatedContent: string, customRegener
 };
 export const rulesBlockTemplate = (source: string, entries: string[]): string => {
   return stripIndents`
-  # Rule${entries.length > 1 ? 's' : ''} extracted from ${source}
+  # Rule${entries.length > 1 ? 's' : ''} extracted from ${source}${
+    entries.some((rule) => rule.trim().includes(' ')) ? '' : '(containing path exclusions)'
```

Comment:
Is this comment needed? If so, I would like there to be a space before the opening parenthesis.

Reply:
yea, it is needed IMO. This is a confusing new addition and is gonna lead to confusing results. So I decided to add a tiny change so ppl will know that it has an exclusion.

Reply:
I see what you mean about the space. It will be so :)